tests/components/matter/fixtures/nodes/contact-sensor.json

```diff
@@ -541,5 +648,9 @@
     }
   },
   "endpoints": [0, 1],
+  "root_device_type_instance": null,
```

Comment:
I added these lines here as I saw them in the light fixture but it wasn't required to make the tests pass, I think.

Reply:
you can leave them out indeed